apps/src/templates/rubrics/rubrics.module.scss

```diff
@@ -110,4 +115,64 @@
 .learningGoalExpanded {
   border-bottom: 1px solid $neutral_dark40;
   padding: 20px 16px;
+
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+  align-self: stretch;
+}
+
+.learningGoalTips {
+  padding: 8px;
+  border-radius: 4px;
+  border: 1px solid $neutral_dark50;
+  background: $neutral_dark10;
+
+  // This is body three styles but the typography components don't work well with markdown support
+  // SafeMarkdown also wraps everything in its own div so explicitly apply this style to the div
+  div {
+    color: $neutral-dark;
+    font-family: $gotham-regular, sans-serif;
+    font-size: 0.875rem;
+    line-height: 1.54;
+  }
+}
+
+.evidenceLevelSet {
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  gap: 16px;
+  align-self: stretch;
+}
+
+.evidenceLevelOption {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 4px;
+  align-self: stretch;
+
+  span {
+    font-size: 14px;
```

Comment:
We agreed to use `rem` instead of `px` for the font size. (and also we can use `rem` for paddings/gaps/margins)
Please use `rem` here and in all other places in this pr